### DISCUSSION TITLE
Reuse of the new INSPIRE logo

### DISCUSSION THREAD

**Discussion by hogredan** [INSPIRE-MIF/helpdesk#176] [Q&A] [answered]
We would like to reuse the new INSPIRE logo for our national documents and applications. Is this possible in principle and is the logo downloadable in an appropriate format? If yes, where is it available?

Best regards,
Daniela

**Comment by jescriu** [comment]
Dear @hogredan,
Find attached the new logo.

We will make sure to offer it through a suitable repository for downloading it - To be discussed with the team.

![LOGO-WEB](https://github.com/INSPIRE-MIF/helpdesk/assets/22743378/030bf3de-5740-4089-b300-6fc66f639ba6)

**Comment by hogredan** [comment]
Dear @jescriu ,

thanks a lot for providing the logo. Is it also available in svg, jpg and/or eps format?

**Comment by jescriu** [comment, marked as answer]
It is available in .pdf and .svg. Attached below.
[LOGO.pdf](https://github.com/INSPIRE-MIF/helpdesk/files/14110983/LOGO.pdf)
![NEW LOGO](https://github.com/INSPIRE-MIF/helpdesk/assets/22743378/54d0896e-e6cd-4479-9530-9692a69325dc)

You can expert .jpg from the .png file.
